src/payment/strategies/paypal-commerce/paypal-commerce-overlay.ts

```diff
@@ -0,0 +1,42 @@
+import { Overlay, OverlayShowOptions } from '../../../common/overlay';
+
+export const idOverlay = 'paypal-commerce-overlay';
+
+export default class OverlayPaypal extends Overlay {
+
+    constructor() {
+        super({ id: idOverlay });
+    }
+
+    show(options?: OverlayShowOptions) {
+        super.show(options);
+        const element = document.getElementById(idOverlay);
+
+        if (element) {
+            const modal = document.createElement('div');
+            const text = document.createElement('div');
+            const link = document.createElement('a');
+
+            modal.className = 'paypal-commerce-overlay_modal';
+            modal.style.position = 'fixed';
+            modal.style.top = '50%';
+            modal.style.left = '50%';
+            modal.style.transform = 'translate(-50%, -50%)';
+            modal.style.color = 'white';
+            modal.style.maxWidth = '330px';
+            modal.style.fontSize = '1.3em';
+
+            text.className = 'paypal-commerce-overlay_text';
+            text.innerText = 'Don\'t see the secure PayPal browser? We\'ll help you re-launch the window to complete your flow. You might need to enable pop-ups in your browser in order to continue.';
```

Comment:
is there a way to provide a way to customize this text? Might be part of `options`. Its quite likely that it might need to be translated.